### ISSUE TITLE
[Bug] NullReferenceException using preview broker with MSA account and MSA-PT enabled for ATS

### ISSUE THREAD

**Issue by mjcheetham** [AzureAD/microsoft-authentication-library-for-dotnet#3743]
**Logs and network traces**

<details>
  <summary>MSAL Logs</summary>

```log
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] MSAL MSAL.Desktop with assembly version '4.47.2.0'. CorrelationId(20e6084f-19cb-40c1-a2ef-3a30b2110dd2)
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] === AcquireTokenSilent Parameters ===
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] LoginHint provided: True
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Account provided: False
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] ForceRefresh: False
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] 
=== Request Data ===
Authority Provided? - True
Scopes - 499b84ac-1321-427f-aa17-267ca6975798/.default
Extra Query Params Keys (space separated) - 
ApiId - AcquireTokenSilent
IsConfidentialClient - False
SendX5C - False
LoginHint ? False
IsBrokerConfigured - True
HomeAccountId - False
CorrelationId - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2
UserAssertion set: False
LongRunningOboCacheKey set: False
Region configured: 

[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] === Token Acquisition (SilentRequest) started:
	 Scopes: 499b84ac-1321-427f-aa17-267ca6975798/.default
	Authority Host: login.microsoftonline.com
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Cache Session Manager] Entering the cache semaphore. Real semaphore: True. Count: 1
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Cache Session Manager] Entered cache semaphore
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z] [Microsoft.Identity.Client.Extensions] Before access
Acquiring lock for token cache
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z] [Microsoft.Identity.Client.Extensions] Before access, the store has changed
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z] [Microsoft.Identity.Client.Extensions] Read '5168' bytes from storage
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z] [Microsoft.Identity.Client.Extensions] Deserializing the store
[Always] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z] [Internal cache] Clearing user token cache accessor.
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z] [Microsoft.Identity.Client.Extensions] After access
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Cache Session Manager] Released cache semaphore
[Always] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z] [Internal cache] Total number of cache partitions found while getting refresh tokens: 0
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [GetAccounts] Found 0 RTs and 1 accounts in MSAL cache. 
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] IsLegacyAdalCacheEnabled: yes
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Region discovery] Not using a regional authority. 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Instance Discovery] Tried to use network cache provider for login.microsoftonline.com. Success? False. 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Instance Discovery] Tried to use known metadata provider for login.microsoftonline.com. Success? True. 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [GetAccounts] Found 0 RTs and 1 accounts in MSAL cache after environment filtering. 
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] IsLegacyAdalCacheEnabled: yes
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Region discovery] Not using a regional authority. 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Instance Discovery] Tried to use network cache provider for login.microsoftonline.com. Success? False. 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Instance Discovery] Tried to use known metadata provider for login.microsoftonline.com. Success? True. 
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] IsLegacyAdalCacheEnabled: yes
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Attempting to acquire token using using local cache...
[Always] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Internal cache] Total number of cache partitions found while getting access tokens: 1
[Always] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [FindAccessTokenAsync] Discovered 1 access tokens in cache using partition key: 00000000-0000-0000-df8c-729242b65de8.9188040d-6c67-4c5b-b112-36a304b66dad
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Filtering AT by tenant id - item count before: 1 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Filtering AT by tenant id - item count after: 0 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Filtering AT by home account id - item count before: 0 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Filtering AT by home account id - item count after: 0 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Filtering by token type - item count before: 0 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Filtering by token type - item count after: 0 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Not filtering by scopes, because there are no candidates
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Not filtering AT by environment, because there are no candidates
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [FindAccessTokenAsync] No tokens found for matching authority, client_id, user and scopes. 
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Region discovery] Not using a regional authority. 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Instance Discovery] Tried to use network cache provider for login.microsoftonline.com. Success? False. 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Instance Discovery] Tried to use known metadata provider for login.microsoftonline.com. Success? True. 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [FOCI] App is not part of the family, skipping FOCI. 
[Always] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z] [Internal cache] Total number of cache partitions found while getting refresh tokens: 0
[Always] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [FindRefreshTokenAsync] Discovered 0 refresh tokens in cache using key: 00000000-0000-0000-df8c-729242b65de8.9188040d-6c67-4c5b-b112-36a304b66dad
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [FindRefreshTokenAsync] No RTs found in the MSAL cache 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [FindRefreshTokenAsync] Checking ADAL cache for matching RT. 
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] IsLegacyAdalCacheEnabled: yes
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Region discovery] Not using a regional authority. 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Instance Discovery] Tried to use network cache provider for login.microsoftonline.com. Success? False. 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] [Instance Discovery] Tried to use known metadata provider for login.microsoftonline.com. Success? True. 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] No Refresh Token was found in the cache. 
[Warning] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Refreshing the RT failed. Is the exception retryable? False. Is there an AT in the cache that is usable? False 
[Warning] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Failed to refresh the RT and cannot use existing AT (expired or missing). 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Token cache could not satisfy silent request.
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Attempting to use broker instead. 
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z] [WamBroker] WAM supported OS.
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z] [WAM Broker] MsalRuntime initialization successful.
[Info] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Can invoke broker. Will attempt to acquire token with broker. 
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z] [WamBroker] Acquiring token silently.
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z] [WamBroker] Validating Common Auth Parameters.
[Verbose] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z] [WamBroker] Acquired Common Auth Parameters.
[Error] False MSAL 4.47.2.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-10-11 21:36:04Z - 20e6084f-19cb-40c1-a2ef-3a30b2110dd2] Exception type: System.NullReferenceException

   at Microsoft.Identity.Client.Broker.RuntimeBroker.<AcquireTokenSilentAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Identity.Client.Internal.Requests.BrokerSilentStrategy.<SendTokenRequestToBrokerAsync>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Identity.Client.Internal.Requests.BrokerSilentStrategy.<ExecuteAsync>d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Identity.Client.Internal.Requests.Silent.SilentRequest.<ExecuteAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Identity.Client.Internal.Requests.RequestBase.<RunAsync>d__12.MoveNext()

```

</details>

**Which version of MSAL.NET are you using?**

Package|Version
-|-
MS.ID.Client|4.47.2
MS.ID.Client.Broker|4.47.2-preview
MS.ID.Client.Extensions.Msal|2.24.0

**Platform**

.NET Framework 4.7.2 or .NET 6 (both repro the same)

Windows 11 21H2 (build 22000.1098)

**What authentication flow has the issue?**
* Desktop / Mobile
    * [x] Silent

**Is this a new or existing app?**

Git Credential Manager; testing the new "native" broker preview.

**Repro**

<details>
  <summary>Repro Code</summary>

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net6.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.Identity.Client" Version="4.47.2" />
    <PackageReference Include="Microsoft.Identity.Client.Broker" Version="4.47.2-preview" />
    <PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Version="2.24.0" />
  </ItemGroup>
</Project>
```

```csharp
using System.Runtime.InteropServices;
using Microsoft.Identity.Client;
using Microsoft.Identity.Client.Broker;
using Microsoft.Identity.Client.Extensions.Msal;

const int GA_ROOTOWNER = 3;

[DllImport("user32.dll", ExactSpelling = true)]
static extern IntPtr GetAncestor(IntPtr hwnd, int flags);

[DllImport("kernel32.dll")]
static extern IntPtr GetConsoleWindow();

IntPtr GetParentWindowHandle() => GetAncestor(GetConsoleWindow(), GA_ROOTOWNER);

const string authorityUrl = "https://login.microsoftonline.com/organizations";

// Git Credential Manager
const string clientId = "d735b71b-9eee-4a4f-ad23-421660877ba6";
const string redirectUrl = "http://localhost";

const string azureDevOps = "499b84ac-1321-427f-aa17-267ca6975798";
string[] scopes = {$"{azureDevOps}/.default"};

bool useMsaPt = true;

var app = PublicClientApplicationBuilder.Create(clientId)
	.WithAuthority(authorityUrl)
	.WithRedirectUri(redirectUrl)
	.WithParentActivityOrWindow(GetParentWindowHandle)
	.WithBrokerPreview()
	.WithWindowsBrokerOptions(
		new WindowsBrokerOptions { MsaPassthrough = useMsaPt }
	)
	.Build();

// Set up token cache (can be empty for repro)
string cacheDirectory = Path.Combine(
	Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
	".IdentityService"
);
var cacheProps = new StorageCreationPropertiesBuilder("msal.cache", cacheDirectory).Build();
var cacheHelper = await MsalCacheHelper.CreateAsync(cacheProps);
cacheHelper.RegisterCache(app.UserTokenCache);

// Populate the cache with at least one user
// At WAM prompt.. sign in with an MSA
var result1 = await app.AcquireTokenInteractive(scopes).ExecuteAsync();
Console.WriteLine(result1.AccessToken);

string loginHint = result1.Account.Username;
Console.WriteLine("Username: {0}", loginHint);

try
{
	// Fails with NRE
	var result2 = await app.AcquireTokenSilent(scopes, loginHint).ExecuteAsync();
	Console.WriteLine(result2.AccessToken);
}
catch (MsalUiRequiredException ex)
{
	Console.WriteLine("UI Required!");
	Console.WriteLine(ex);
}
```

</details>

**Expected behavior**
Silent authentication succeeds.

**Actual behavior**
`NullReferenceException` is thrown.

**Possible solution**
Unknown.

**Additional context / logs / screenshots / links to code**

- The GCM client ID is a 1st Party Microsoft AAD application
- At the WAM sign-in dialog, I select "Personal account" and sign in with an MSA
- At the post-sign-in dialog where it asks to "remember this account" (or words to that affect), I clicked "Microsoft Apps Only"
- AAD accounts work fine with ATS
- Building for `win-x86`

**Comment by bgavrilMS** [comment]
@SameerK-MSFT - please try to repro and root cause this.

**Comment by gladjohn** [comment]
This works when we pass the IAccount in the ATS call, instead of the loginHint. Our test app blocks this with this message, which hides the actual issue 

![image](https://user-images.githubusercontent.com/90415114/195761729-d31d5f39-0020-4f4c-8212-954ceff23954.png)

**Comment by mjcheetham** [comment]
> This works when we pass the IAccount in the ATS call, instead of the loginHint. Our test app blocks this with this message, which hides the actual issue

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/479f5afeeeb74a39078e63972d8fa5cc37f35be3/tests/devapps/WAM/NetFrameworkWam/Form1.cs#L182-L195

The `TODO` here seems to imply the combination is something that shouldn't be supported and should be actively blocked - is there a reason for that? (MSA-PT + login hint)

**Comment by gladjohn** [comment]
@mjcheetham this is fixed in MSAL 4.48.0
